SCAutolib/__init__.py

```diff
@@ -18,141 +15,16 @@
 logging.getLogger("invoke").setLevel(logging.WARNING)
 logging.getLogger("fabric").setLevel(logging.WARNING)
 
-DIR_PATH = dirname(abspath(__file__))
-TEMPLATES_DIR = Path(DIR_PATH, "templates")
-
-SETUP_IPA_SERVER = f"{DIR_PATH}/env/ipa-install-server.sh"
-LIB_DIR = "/etc/SCAutolib"
-CONF = join(LIB_DIR, "user-conf.yaml")
-LIB_CONF = join(LIB_DIR, "lib-conf.yaml")
-LIB_CA = join(LIB_DIR, "ca")
-LIB_BACKUP = join(LIB_DIR, "backup")
-LIB_KEYS = join(LIB_DIR, "keys")
-LIB_CERTS = join(LIB_DIR, "certs")
-
-
-def init_config(user_config=None, config_content: dict = None):
```

Comment:
Why were `init_config`, `read_config`, and `set_config` methods removed since the last review? Is it related to something I said?

Reply:
Yes, it is connected. These functions are irrelevant for a new version of the library. In [this](https://github.com/x00Pavel/SCAutolib/pull/98#discussion_r930016061) comment you are asking about some variable that are using `.join()` method. When I removed those variables, there were several problems with unexisting variables in these irrelevant functions. So I remove them also.

Reply:
Well I meant changing `join` to `joinpath`, not removing those variables altogether. So they were not needed anyway?

Reply:
yes, they are not needed anymore as those functions that were using these variables